components/dashboard/src/App.tsx

```diff
@@ -64,7 +64,7 @@ function App() {
     }
     if (window.location.pathname.startsWith('/blocked')) {
         return <div className="mt-48 text-center">
-            <img src={gitpodIcon} className="h-16 mx-auto"/>
+            <img src={process.env.REACT_APP_ICON != undefined ? gitpodHostUrl.toString() + process.env.REACT_APP_ICON : gitpodIcon} className="h-16 mx-auto"/>
```

Comment:
This is done because by the time react environment variables are replaced the icons folder from where gitpodIcon is fetched doesn't exist.

Reply:
why do we need to add this gitpodHostUrl.toString(). This is forcing us to keep the files with in the build folders.

process.env.REACT_APP_ICON != undefined ?  process.env.REACT_APP_ICON : gitpodIcon

Reply:
That's not what's happening here. We are just accessing the public static files that are stored.The gitpodHostUrl resolves to the host URL of the deployment.

Reply:
we can remove that but we'll have to include the host URL in the .env file